imls-backend/db/migrations/20220912214016_create_update_presemce_func.sql

```diff
@@ -0,0 +1,32 @@
+-- migrate:up
+-- FUNCTION: api.update_presence(timestamp with time zone, timestamp with time zone, character varying, integer, integer)
+
+CREATE OR REPLACE FUNCTION api.update_presence(
+	_start timestamptz,
+	_end timestamptz,
+	_fscs character varying (16),
+	_sensor integer,
+	_manufacture integer)
+    RETURNS character varying
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE PARALLEL UNSAFE
```

Comment:
Does this mean we can't have concurrent calls to this function (for now)? Not a blocker, just wondering if we should make this explicit in the documentation.